src/client.ts

```diff
@@ -1,118 +1,87 @@
 import * as VError from 'verror'
-import { Cache } from './cache'
-import { ClientLowLevel } from './clientLowLevel'
+import axios, { AxiosStatic } from 'axios'
 import Debug from './debug'
-const debug = Debug.extend('high')
+
+const debug = Debug.extend('client')
 
 interface Config {
-  cache?: Cache
-  retries: number
-  lowLevelClient: ClientLowLevel
+  uri: any
```

Comment:
Would be good to fina a more descriptive name for this, e.g. `serverEndpoint`, `serverUrl`, `serverAddress`, ...

Reply:
made it `serverUri`